omegaconf/grammar/OmegaConfGrammarParser.g4

```diff
@@ -41,7 +41,7 @@ sequence: (element (COMMA element?)*) | (COMMA element?)+;
 // Interpolations.
 
 interpolation: interpolationNode | interpolationResolver;
-interpolationNode: INTER_OPEN DOT* configKey (DOT configKey)* INTER_CLOSE;
+interpolationNode: INTER_OPEN DOT* (configKey (DOT configKey)*)? INTER_CLOSE;
```

Comment:
OmegaConf select supports passing an empty string to select the root of the config:
```python
In [5]: cfg = OmegaConf.create({"foo": 10})

In [6]: OmegaConf.select(cfg, "foo")
Out[6]: 10

In [7]: OmegaConf.select(cfg, "")
Out[7]: {'foo': 10}
```

Maybe we should make ${} mean the root node instead of failing.

Reply:
> Maybe we should make `${}` mean the root node instead of failing.

Sounds like a good idea to me.

Reply:
Done: 0446b93